src/index.js

```diff
@@ -7,16 +7,14 @@ import 'jquery/src/jquery';
 import 'bootstrap/dist/js/bootstrap.min';
 
 import React from 'react';
-import ReactDOM from 'react-dom';
+import { createRoot } from 'react-dom/client';
 import './index.css';
 import App from './App';
 import { unregister } from './registerServiceWorker';
 import { BrowserRouter } from 'react-router-dom';
 
 unregister();
 
```

Comment:
Actually I did notice an error in my terminal - it says compiled with 1 warning:
`Failed to parse source map from '/Users/qhoque/GIT/duos-ui/node_modules/paging-algorithm/src/index.ts' file: Error: ENOENT: no such file or directory, open '/Users/qhoque/GIT/duos-ui/node_modules/paging-algorithm/src/index.ts'

WARNING in ./node_modules/paging-algorithm/dist/index.js
Module Warning (from ./node_modules/source-map-loader/dist/cjs.js):
Failed to parse source map from '/Users/qhoque/GIT/duos-ui/node_modules/paging-algorithm/src/index.ts' file: Error: ENOENT: no such file or directory, open '/Users/qhoque/GIT/duos-ui/node_modules/paging-algorithm/src/index.ts'
`

I did not see the same warning when running locally from develop.

Reply:
I've left a comment on it above